inc/drivers/cutebot/CutebotLineSensors.h

```diff
@@ -0,0 +1,24 @@
+#ifndef CUTEBOT_LINE_SENSORS_H
+#define CUTEBOT_LINE_SENSORS_H
+
+#include "MicroBitComponent.h"
+#include "MicroBitIO.h"
+
+class CutebotLineSensors : public MicroBitComponent
+{
+    MicroBitIO io;
```

Comment:
Should this be `&io`?

Reply:
Yes. I'll fix that in the GigglebotLineSensors.h, too.

Reply:
(for the Gigglebot's i2c)